### PR TITLE
[Grouping&Mapping] Replaced mapping table

### PR DESCRIPTION
**Pull request by arinsasmal**

### CODE REVIEW

File: packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx [hunk 1]
```diff
@@ -14,24 +14,22 @@ import {
...
+import { EmptyMessage, handleError, LoadingOverlay, onSelectionChanged, WidgetHeader } from "./utils";
 import "./Mapping.scss";
 import DeleteModal from "./DeleteModal";
 import { Groupings } from "./Grouping";
 import MappingAction from "./MappingAction";
 import { MappingImportWizardModal } from "./MappingImportWizardModal";
 import { useMappingClient } from "./context/MappingClientContext";
 import type { Mapping } from "@itwin/insights-client";
-import { BlockingOverlay } from "./BlockingOverlay";
+import { HorizontalTile } from "./HorizontalTile";
```
Comment (Guillar1) [inline packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx:32]: Can you bring back the blocking overlay?

Comment (arinsasmal) [reply packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx:32]: Will do.

File: packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx [hunk 2]
```diff
@@ -238,30 +146,100 @@ export const Mappings = () => {
...
-                <IconButton title="Import Mappings">
+              <div className = "gmw-button-spacing">
+                <Button
+                  startIcon={<SvgAdd />}
+                  onClick={async () => addMapping()}
+                  styleType="high-visibility"
+                >
+                  New
+                </Button>
+
```
Comment (Guillar1) [inline packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx:159]: Random line.

File: packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx [hunk 3]
```diff
@@ -238,30 +146,100 @@ export const Mappings = () => {
...
+                              key={0}
+                              onClick={() => {
+                                setSelectedMapping(mapping);
+                                setMappingView(MappingView.MODIFYING);
+                              }}
+                              icon={<SvgEdit />}
+                            >
+                              Modify
+                            </MenuItem>,
+
```
Comment (Guillar1) [inline packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx:198]: Random line.

File: packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx [hunk 4]
```diff
@@ -238,30 +146,100 @@ export const Mappings = () => {
...
+                          <IconButton styleType="borderless">
+                            <SvgMore
+                              style={{
+                                width: "16px",
+                                height: "16px",
+                              }}
+                            />
+                          </IconButton>
+                        </DropdownMenu>
+
```
Comment (Guillar1) [inline packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx:236]: Here too.

File: packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx (hunk 2 above)
Comment (arinsasmal) [reply packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx]: Done

File: packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx (hunk 3 above)
Comment (arinsasmal) [reply packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx]: Done

File: packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx (hunk 4 above)
Comment (arinsasmal) [reply packages/itwin/grouping-mapping-widget/src/widget/components/Mapping.tsx]: Done

**Review by Guillar1 — APPROVED** [review]

**Pull request merged**